presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -155,6 +157,18 @@ public int run(List<QueryPair> queries)
         }
 
         log.info("Results: %s / %s (%s skipped)", valid, failed, skipped);
+        log.info("");
+
+        for (EventClient eventClient : eventClients) {
+            if (eventClient instanceof Closeable) {
+                try {
+                    ((Closeable) eventClient).close();
+                }
+                catch (IOException ignored) { }
+                log.info("");
```

Comment:
This is here to cause a blank line between each client and a blank line before the end of the file.... basically a repeat of the log line on 160 above